mbof/views.py

```diff
@@ -26,6 +27,15 @@ class EventViewSet(viewsets.ModelViewSet):
     """
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
+    
+
+class CurrentEventViewSet(EventViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
+    """
+    current_time = timezone.now()
+    queryset = Event.objects.filter(startTime__lte=current_time).filter(endTime__gte=current_time)\
```

Comment:
OK, sorry about putting so many comments in one place.  😆 

I just realized:  This query needs to use the current time, right?  Didn't we find that it won't work if you assign it to
the class' `queryset` attribute, it needs to be defined in a `get_queryset()` method instead?